src/drive/mobile/containers/UploadProgression.jsx

```diff
@@ -54,20 +54,38 @@ const UploadUptodate = ({ t }) => (
   </div>
 )
 
+const UploadAbortedWifi = ({ t }) => (
+  <div
+    className={classnames(
+      styles['coz-upload-status'],
+      styles['coz-upload-status--waiting']
+    )}
+  >
+    <div className={styles['coz-upload-status-content']}>
+      {t('mobile.settings.media_backup.media_wifi')}
+    </div>
+  </div>
+)
+
 const mapStateToProps = state =>
   state.mobile.mediaBackup.currentUpload
     ? {
         media: state.mobile.mediaBackup.currentUpload.media,
         current: state.mobile.mediaBackup.currentUpload.messageData.current,
         total: state.mobile.mediaBackup.currentUpload.messageData.total
       }
-    : { media: undefined, current: undefined, total: undefined }
+    : {
+        media: undefined,
+        current: undefined,
+        total: undefined,
+        aborted: state.mobile.mediaBackup.abortedMediaBackup
+      }
 
-const UploadStatus = props =>
-  props.media && props.current && props.total ? (
-    <UploadProgression {...props} />
-  ) : (
-    <UploadUptodate {...props} />
-  )
+const UploadStatus = props => {
+  if (props.media && props.current && props.total)
+    return <UploadProgression {...props} />
+  else if (props.aborted) return <UploadAbortedWifi {...props} />
```

Comment:
Pas besoin de passer toutes les props, car `<UploadAbortedWifi />` n'a besoin que de `t`.

Reply:
C'est le cas sur les autres composants, c'est vrai. Et je dois même être l'auteur. Comme quoi !
Après tout, c'est plus simple, bien que comme dit Guido
> explicit is better than implicit

Reply:
J'ai gardé le même format pour l'homogénéité, mais je me suis fait la réflexion aussi. Si ca te va je changerais bien ca pour passer à des props explicites sur tous ces composants.